traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -107,6 +107,7 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		//CDN: CRUD
 		{1.1, http.MethodGet, `cdns/?(\.json)?$`, api.ReadHandler(cdn.GetRefType(), d.DB), auth.PrivLevelReadOnly, Authenticated, nil},
 		{1.1, http.MethodGet, `cdns/{id}$`, api.ReadHandler(cdn.GetRefType(), d.DB), auth.PrivLevelReadOnly, Authenticated, nil},
+		{1.1, http.MethodGet, `cdns/name/{name}$`, cdn.GetName(d.DB.DB), auth.PrivLevelReadOnly, Authenticated, nil},
```

Comment:
@rob05c - can you change this to use the generic ReadHandler like Dylan said? We need to stay within the parameters of the framework when possible imo.